source/ways-of-working/standards/pull-requests.html.md.erb

```diff
@@ -130,6 +130,19 @@ All repositories in the organisation will automatically use this by default.
 
 You should start with this and adopt for your needs as needed by copying it into repository in the `.github` folder and editing it.
 
+### Suggested Branch Protection Rules
+
+These are the recommended settings to configure for your main branch on your repository, however each project might run differently so these are not These are the recommended settings to configure for your main branch on your repository, however each project might run differently so these are not compulsory.
+Check and fill out the below options on the Branch Protection Rules of your main branch.
+
+- Require a pull request before merging
+    - Require Approvals (minimal 1)
+    - Dismiss stale pull request approvals when new commits are pushed
```

Comment:
generally shows a distrust of team, reviews should be re-requested if substantially changes, not require re-checking every time.
I recommend this should only be used on repositories that grant access to something or e.g. flux-config that
```suggestion
```

Reply:
If I leave this in and add a note in brackets for :
`Only use this for security related repositories and repositories shared across multiple teams`

Reply:
No, if you really want to keep it put it in a different separated section that is clearly caveated

people may not read to the end and see the brackets

Reply:
ok removed